### PR TITLE
feat(topbar/user): allo setting notifications in the cta

### PR DESCRIPTION
**Pull request by naxhh**

### CODE REVIEW

File: components/topbar/user/src/index.scss [hunk 1]
```diff
@@ -118,6 +119,19 @@
             fill: $c-primary !important;
           }
         }
+
+        &Notifications {
+          background-color: $c-primary;
+          border-radius: 50%;
+          color: $c-white;
+          font-size: $fz-xs;
+          height: 17px;
```
Comment (kikoruiz) [inline components/topbar/user/src/index.scss]: This magic number is asking for a Sass var 😝

File: components/topbar/user/src/index.scss [hunk 2]
```diff
@@ -147,6 +161,21 @@
             $background-color: $bgc-topbar-user-cta,
             $icon-fill: $c-white
           );
+          position: relative;
+        }
+
+        &Notifications {
+          background-color: $c-primary;
```
Comment (kikoruiz) [inline components/topbar/user/src/index.scss]: I think we can reuse some common code and create a mixin with it, then apply specific styles in each selector. Can help if you need.

File: components/topbar/user/src/index.scss [hunk 3]
```diff
@@ -147,6 +161,21 @@
...
+          border-radius: 50%;
+          color: $c-white;
+          font-size: $fz-xs;
+          height: 17px;
+          line-height: 17px;
+          position: absolute;
+          right: -7;
```
Comment (midudev) [inline components/topbar/user/src/index.scss]: Missing the `px`here and in the top. :+1:

Comment (naxhh) [reply components/topbar/user/src/index.scss]: ups!

File: components/topbar/user/src/index.scss (hunk 1 above)
Comment (naxhh) [reply components/topbar/user/src/index.scss]: done!

File: components/topbar/user/src/index.scss (hunk 2 above)
Comment (naxhh) [reply components/topbar/user/src/index.scss]: and done!

I don't know if what i did is what you wanted, or just keep top & right outside the mixin?

**Review by kikoruiz — APPROVED** [review]
👏

**Review by JoanClaret — APPROVED** [review]
Like a lot the mixin solution to avoid duplicated code. congratulations!

**Review by carlosvillu — APPROVED** [review]

**Review by carlosvillu — APPROVED** [review]

**Pull request merged**